README.md

```diff
@@ -17,8 +17,22 @@ You can debug your code, quickly run `react-native` commands from the command pa
   1. Run `npm install -g react-native-cli` to install React Native CLI (0.1.10+)
   2. Set up React Native (0.19+) using the steps detailed on the React Native [getting started documentation ](https://facebook.github.io/react-native/docs/getting-started.html)
 * Open your React Native project root folder in VS Code.
-
-## Debugging
+  * Hint: you should create a `.babelrc` with `sourceMaps = true` and `"presets": [ "react-native" ]` for a better source mapping. (**required if want TypeScript supports**)
```

Comment:
I'd replace:
"for a better source mapping"
for
"for better source-mapping support"